src/Umbraco.Core/EmbeddedResources/Lang/da.xml

```diff
@@ -2094,15 +2094,15 @@ Mange hilsner fra Umbraco robotten
   <area alias="references">
     <key alias="tabName">Referencer</key>
     <key alias="DataTypeNoReferences">Denne Data Type har ingen referencer.</key>
-    <key alias="labelUsedByMediaTypes">Brugt i Medie Typer</key>
-    <key alias="labelUsedByMemberTypes">Brugt i Medlems Typer</key>
-    <key alias="usedByProperties">Brugt af</key>
-    <key alias="labelUsedByDocuments">Brugt i Dokumenter</key>
-    <key alias="labelUsedByMembers">Brugt i Medlemmer</key>
-    <key alias="labelUsedByMedia">Brugt i Medier</key>
+    <key alias="labelUsedByMediaTypes">Bruges i Medie Typer</key>
+    <key alias="labelUsedByMemberTypes">Bruges i Medlems Typer</key>
```

Comment:
Not a line you've touched, as I'm commenting on it since it adjacent to two lines you changed. `Denne Data Type har ingen referencer.` should IMO be `Denne datatype har ingen referencer.` instead.

`Bruges i Medie Typer` should also be `Bruges i Medietyper` and `Bruges i Medlems Typer` should be `Bruges i Medlemstyper`.

Reply:
👍